### **Cellular Reprogramming as a Rejuvenative Therapy**

**The Role of DNA Methylation in Anti-Ageing Research and the Therapeutic Efficacy of Rejuvenative Reprogramming**

**Blind Grading Number:** 7967R

**Supervisor:** Michael Imbeault

**Word Count:** 5998

---

### **Summary**

The Information Theory of Ageing states that all the deleterious age-associated phenotypes people experience have a single root cause: the breakdown of the epigenome over time. This theory is supported by the efficacy of methylation clocks, which operate based on a correlation between the chronological age of a person and the epigenetic changes to their methylome over time. In combination with the rejuvenative success seen with epigenetic reprogramming technologies in recent years, this theory of ageing causality is at the forefront of current research.

The discovery of DNA methylation clocks is based on this correlation between epigenetic age and chronological age. I will explore the analytical utility DNA methylation clocks have in accelerating anti-ageing research and discuss the direct role the methylome, as a component of the epigenome, could play during the ageing process.

Completely rejuvenated, pluripotent cells can be created through Yamanaka Factor treatment. The challenge for any therapeutically viable rejuvenative treatment is to decouple the rejuvenative effects of Yamanaka Factor treatment from its dedifferentiative effects. Historically, the testing of the effects of these candidate treatments on cells has been limited by a reliance on bulk-cell transcriptomic and epigenetic data. These experimental procedures do not show whether there are small numbers of cells becoming induced pluripotent stem cells (iPSCs) after treatment, due to their dependence on bulk-cell derived data. However, the retention of pluripotency in any of the cells undergoing a rejuvenative treatment will have disastrous tumorigenic consequences. Due to the advent of single-cell transcriptomic and epigenomic analysis, scientists will now be able to more rigorously test any candidate rejuvenative treatments, and attempt to correct any faults they have.

I will aim to explain what obstacles these therapies must overcome, and the direction of current research.

---

### **Contents**

- **Summary**
    
- **Contents**
    
- **Introduction**
    
- **2 - DNA Methylation**
    
    - 2.1 - The Basics of DNA Methylation
        
    - 2.2 - Methylation Clocks
        
    - 2.3 - The Information Theory of Ageing
        
    - 2.4 - The Methylation Cycle during Embryogenesis
        
    - 2.5 - Differentially Methylated Regions
        
    - 2.6 - Age-Dependent Gain of Methylation
        
    - 2.7 - Age-Dependent Loss of Methylation
        
    - 2.8 - Experimental Technique and DMR Masking
        
    - 2.9 - DNA Methylation: Physical Time or a Wristwatch?
        
- **3 - Reprogramming as a Rejuvenative Tool**
    
    - 3.1 - The Yamanaka Factors
        
    - 3.2 - Teratoma Formation
        
    - 3.3 - Cell Type Specific Reprogramming Responses
        
- **Conclusion**
    
- **Bibliography**
    

---

### **Acknowledgement**

I would like to thank Michael, for our regular supervisions and thought provoking discussions.

---

### **Introduction**

The idea of nonexistence is unfathomable, and to many, terrifying. To the average person, attempting to cheat death is a ludicrous fantasy, better kept between the covers of a science-fiction book, than discussed in front of a star-studded scientific panel.

But as they say “young-people dream of being rich, while rich people dream of being young”. The super rich have never been bound by the rules of normal society. Jeff Bezos and Yuri Milner are two of the wealthiest people in the world, with a combined net worth exceeding Hungary’s GDP, and recently they have both invested large sums of money into the founding of Altos Labs. This private startup is focussed on tackling the ageing process, with many notable experts in the field such as Shinya Yamanaka and Steve Horvath involved in its inception. Although this is the latest in a series of efforts over the past decade at a coordinated attack on the ageing process, recent breakthroughs with methylation clocks and single-cell analytics may shed light on whether immortality really does have a place in reality.

The cellular ageing process has technologies that has slowly progressed over the past decade and may have an explanation of the ageing process. Although this does have a series of effects on the cellular ageing process, with many notable experts in the field such as Shinya Yamanaka and Steve Horvath involved in its inception. Although this is the latest in a series of efforts over the past decade at a coordinated attack on the ageing process, recent breakthroughs with methylation clocks and single-cell analytics may shed light on whether immortality really does have a place in reality.

The Information Theory of Ageing states that this ageing process is a consequence of a breakdown of its epigenome over time. All age-associated deleterious effects can be blamed on the breakdown of its epigenome. One of the main avenues through which modern scientists have been able to explore this is through the development of methylation clocks. As anti-ageing researcher David Sinclair puts it in his book, this is the “difference between a wristwatch and physical time” (Sinclair & LaPlante, 2019). These are a series of sites from the DNA methylome whose methylation status can be used as an analogue of chronological age. There is debate surrounding whether the methylome is merely a predictive biological age for the better part of a decade now, as such as from a forefront of many drug testing programmes and clinical trials. There is debate surrounding whether the methylome is merely a predictive biological age for the better part of a decade now, or whether it can be used to prove that ageing is an information process. As anti-ageing researcher David Sinclair puts it in his book, this is the “difference between a wristwatch and physical time” (Sinclair & LaPlante, 2019).

A manipulation of the methylome is also a known way via which cells can modulate their gene expression patterns. And though age-associated changes to the methylome do have effects on gene expression patterns, I will discuss why evidence suggests that the methylome is but one of many components of the epigenome causal to ageing and is simply the easiest to quantify with science’s current technological capabilities. Because of this, a rejuvenative therapy based on cellular reprogramming would aim to wind back a cell’s epigenetic clock, reversing not only the age-associated changes in the DNA methylome, but comprehensively doing the same for the other components of the epigenome responsible for the ageing process.

Yamanaka Factor induced cellular reprogramming appears to rejuvenate the epigenome to the desired effect. With this treatment, normal somatic cells can be transformed into rejuvenated pluripotent stem cells, capable of generating fully functional embryos in mice through tetraploid embryo complementation (TEC) experiments (Boland et al., 2012). The practical use with this technology to rejuvenate ageing humans is the linkage between the dedifferentiation of the somatic cells to pluripotency and their epigenetic rejuvenation. If pluripotency were to be induced by a candidate therapy even in a minute percentage of treated cells, then disastrous tumorigenic consequences would result.

Here we land on an important question about the nature of the ageing epigenome. Perhaps the epigenetic changes that are responsible for cellular ageing have some overlap with those that are needed for a cell to stay differentiated? To date, scientists have never observed a complete rejuvenation with alterations that are currently attempting a separation between dedifferentiation and rejuvenative reprogramming. This is discussed above, there is immense economic and emotional interest in elucidating the nature of these epigenetic changes as this is a critical step to take before any discussion about a complete ageing cure can begin to take place.

This dissertation will be split into two main sections. First, I will discuss the utility of the previously mentioned Information Theory of Ageing, and what the direct effects of the specific changes recorded via methylation clocks may be on the ageing process. In the second section, I will provide an overview of cellular reprogramming as a rejuvenative instrument, discussing the key obstacles preventing it from being therapeutically viable, and the current streams of research directed at tackling these issues.

The seemingly fantastical goal of creating an “ageing cure” is currently driving large resources in the direction of cellular reprogramming technologies. Although there is scepticism on whether it will be possible to develop an anti-ageing therapy, progressing research into anti-ageing will have a multitude of collateral benefits. Previously untreatable diseases may become curable, human tissues may be reprogrammed and grown in labs to supply the organ transplant market, and a partial abatement of age-associated deleterious phenotypes could all stem from further advancements in cellular reprogramming technology.

---

### **2 - DNA Methylation**

#### **2.1 - The Basics of DNA Methylation**

One of the ways in which informational noise manifests itself is as changes to the DNA methylome over time. When a guanine is positioned on the 3’ end of a cytosine base in the DNA, it has a propensity to become methylated in eukaryotes. This two base sequence is called a CpG site, and can be targeted by various methylome modifying enzymes due to the molecular structure of the two bases. DNA methyl-transferase (DNMT) enzymes, using a coenzyme, will add methyl groups to the cytosines of these CpGs. Ten-eleven translocase (TET) enzymes will catalyse an active demethylation reaction, involving the excision of the 5mC and subsequent substitution of an unmethylated cytosine. Although other enzymes have roles in the manipulation of the DNA methylome, these two are the key players.

While the general idea that decreased global DNA methylation is correlated with higher cellular age is statistically true, it is misleading when trying to understand the process on an intimate level. In fact, the specific locus of a CpG will determine the causality of its methylation state. I will discuss how advanced methylation clocks support this idea, to provide scientists with a deep insight into the question of when, and how fast, ageing happens.

#### **2.2 - Methylation Clocks**

In 2013, Steve Horvath invented the first DNA methylation clock able to predict biological age (Horvath, 2013). This methylation clock took a sample of 353 CpG sites in the genome and tested whether they were methylated or not. Using machine learning, algorithms were generated that scored the sites in relation to longevity dependent on their methylation status. This was possible through the feeding of DNA methylomes from old and young patients into a computer. A reading of the methylation status of this tiny proportion of the total CpG sites of the human genome was able to accurately predict biological age. A later iteration of the clock, trained on both chronological age and age-associated phenotypes, named GrimAge, was even able to predict mortality with such efficacy that it was used as the endpoint in clinical trials (Drew, 2022). Many methylation clocks have since been invented and trained on data sets involving cells of differing tissues. The impressive accuracy with which chronological age can be predicted in this manner is indicative of the intimacy between the DNA methylome and the ageing process. Today, methylation clocks are a better predictor of chronological age than either transcriptome analysis or telomere length (Jylhävä et al., 2017).

Methylation clocks have not only promoted DNA methylation to the forefront of ageing research, but have also given scientists the ability to conduct ageing related experiments in a fraction of the time they would otherwise have taken. Scientists can now subject cells to conditions, wait for a week or two, then look for methylome related changes in the cellular age that would have been phenotypically invisible. This has led to notable advancements in the field of anti-ageing research with this biotech sector now attracting record levels of funding (de Magalhães & Ocampo, 2022).

#### **2.3 - The Information Theory of Ageing**

Modern genetics has advanced to the point where scientists have developed overarching arguments for the causality of the process of cellular ageing. The leading theory on why cells deteriorate and senesce over time is inspiration from the Information Theory of Ageing, pioneered by David Sinclair in 2019. This theory takes inspiration from Claude Shannon’s mathematical theory of communication (Shannon, 1948). It can be used as a framework in which these age-associated methylome changes can be mapped to the ageing process.

**Figure 1 - The Information Theory of Ageing (Sinclair & LaPlante, 2019).** For the sake of clarity, I have simplified the Information Theory of Ageing. This distorts the epigenome in the same way a radio communication is distorted when sent through space. Sinclair aborts the epigenome and proliferates in response to changes to the methylome, like those measured with Horvath’s clock. While the blue line represents the complete reversion to pluripotency that comes with iPSC generation, the red line represents the goal of rejuvenative reprogramming. As it currently stands, rejuvenative reprogramming research accomplishes this complete somatic rejuvenation. Horvath’s clock is looking at a subset of loci whose methylation status is affected by the imposition of this “informational noise”. Importantly, these loci are but a minute subset of the changes that will be taking place in the methylome over the course of the ageing process, and the methylome changes are but one component of the epigenetic changes resulting from the ageing process, but a minute subset of the changes that will take place in the epigenome depending on random stochastics and the cell type, but Horvath’s 353 sites seem to be a good pan-tissue indicator of cellular ageing.

#### **2.4 - The Methylation Cycle during Embryogenesis**

The issue of tackling cellular ageing does not seem as far-fetched as one might think when the natural rejuvenative properties of our own bodies are considered. Two 40-year old humans can give birth to a 0-year old human, so logically there must be either an ageing prevention system or a rejuvenative system acting on the parents’ germline. Studies show that there is a rejuvenation event during early embryogenesis, giving rise to the epigenetically “ground zero”. To test this, back to the faithful information in the transmitter of Sinclair’s Information Theory of Ageing.

Kerepesi et al. reported in their 2021 paper on the epigenetic age of mouse embryonic cells using a multitude of methylation clocks trained on a variety of tissues. They also tested the prenatal changes in the epigenetic age of these cells with their own newly developed rDNA multi-tissue clock. An interrogation of the results elucidated a “ground zero” of epigenetic age where the embryonic cells were at their youngest.

**Figure 2 - The Germline Methylation Cycle.** Methylation clocks have found evidence of prenatal ageing occurring during early embryogenesis prior to gastrulation. Here is a schematic of when gastrulation and the ground zero inducing rejuvenation event occur during mouse embryogenesis. Gastrulation occurs at roughly E6.5, after implantation. This is thought of as the initial tissue differentiation event, when embryonic stem cells (ESCs) undergo a loss of pluripotency as they differentiate into all the cells needed to make up the organism. The rejuvenation of the methylome seems to occur in tandem with this gastrulation event. TET enzymes are necessary for successful rejuvenation. TET triple knockout mouse embryos will undergo prenatal death due to defective gastrulation.

Implantation happens at E5.5. This “ground zero” epigenetic state appears to be present between E6.5 and E7.5, which is roughly coincident with the statistical and initial tissue differentiation.

Single-cell methylation clocks like ScAge also allow the use of TET knockout mouse embryos that have allowed scientists to also pinpoint the statistically significant effect TETs have on decreasing the epigenetic age of the early embryo post gastrulation in the nascent tissues (Kerepesi et al., 2021). TET2 and TET3 triple knock-out mouse embryos are not viable, due to their failure to progress through gastrulation and initial tissue formation (Caldwell & Bartolomei, 2022). The ground zero methylome state cannot actually be reached without the active demethylating activity of the TET enzymes during gastrulation. Combined with this, as will be built on later, there is evidence for the importance of the methylome’s contribution to cellular reprogramming process that is also indicative of the methylome’s contribution to cellular age. Although the specific quantification of “ground zero” at a certain embryonic stage is somewhat arbitrary due to inevitable error rates of DNA methylation clocks, the idea of a completely rejuvenated epigenomic state occurring around the same time is notable. The loss of “ground zero” appears to correlate with the loss of complete pluripotency and initial tissue differentiation. So, even when analysing the ageing process right at its beginning, it still holds true that scientists cannot observe a cell that is completely rejuvenated without it being completely pluripotent at the same time.

#### **2.5 - Differentially Methylated Regions**

Much of the recent research in the field of epigenetic rejuvenation is focussed on decoupling the epigenetic changes associated with differentiation from those associated with dedifferentiation (Gill et al., 2022; Roux et. al., 2022; Chondronasiou et al., 2022). From Horvath’s initial pan tissue methylation clocks, various differentially methylated regions (DMRs) can be observed upon a comparison of young and old cells. In fact these regions will not correlate perfectly well with chronological age in all tissues, but have been statistically proven to predict age to an effective extent. If we are to assume that the DMRs involved in Horvath’s clocks are in fact resultant from ageing and are not merely indicative of the changing cell composition of blood and other tissues as ageing occurs, then we can make the following logical steps.

Here are two possibilities as to the nature of methylation changes at DMRs:

1. The changes have no effect on cell differentiation status. They occur as a product of, or drive, the ageing process happening in the cell. Any cell requires an epigenetic age if it is not marked for differentiation. It may be the case that certain DMRs play roles both in the differentiation of the cell and the epigenetic ageing of the cell.
    
2. It may be the case that certain DMRs play roles both in the differentiation of the cell and the epigenetic ageing of the cell. This would be poor news for rejuvenative therapy development as it would be an argument to be made for the plasticity of the cell types and body ages. In response to various hormonal or other chemical signalling, cells will inter their expression profiles and functions within the body. While scientists use to classify cells into discrete groups, it may be the case that cell function and expression patterns exist on a spectrum. So perhaps a continuous micro-manipulation of cell types occurs throughout the ageing process, which may manifest itself in changes to the DNA methylome. This possibility adds to another ambiguity that anti-ageing researchers must wrestle with.
    

A third possibility, that these DMRs are causal to differentiation and independent of ageing, is improbable as these DMRs have been found through a comparison of both young and old differentiated cells. However, there is an argument to be made for the plasticity of cell types as the body ages. In response to various hormonal or other chemical signalling, cells will alter their expression profiles and functions within the body. While scientists use to classify cells into discrete groups, it may be the case that cell function and expression patterns exist on a spectrum. So perhaps a manifest itself in changes to the DNA methylome. This possibility adds to another ambiguity that anti-ageing researchers must wrestle with.

These methylation modifications on the DNA methylome will have effects on the expression levels of proximal genes. Over time and successive cellular divisions, Sinclair says the methylome of the somatic cells will decay as age-associated epigenetic changes on gene expression patterns will jeopardise the differentiated state of somatic cells, causing age-associated phenotypes on surface, from tumorigenic risk and organ failure to decreases in white blood cell counts and wrinkling of the skin.

#### **2.6 - Age-Dependent Gain of Methylation**

The DMRs found through comparisons of young and old cells can be split into two classes: those loci that are hypermethylated due to the ageing process (hyper-DMRs) and those that are hypomethylated (hypo-DMRs). I will first consider the loci that are age-dependently hypermethylated.

**Figure 3 - Example of a developmental gene’s CpG promoter.** The hyper-DMRs are associated with the promoter regions of silent, bivalent genes. As cells differentiate, the TET enzymes no longer active. Therefore PRC2 no longer able to recruit them to the bivalent promoters with the same degree of efficiency. This leaves the promoter unprotected from methylation. The methylation of the promoters causes the silencing off of their associated genes. Horvath postulates that this could be a commitment pathway – the cell is switching off developmental genes in a permanent fashion once differentiation has occurred.

193 of Horvath’s 353 original-clock CpGs are often hypermethylated because of ageing (Horvath & Raj, 2018). Evidence has shown that methylation closely corresponds to transcriptionally silent, poised promoters. Therefore, it is no longer believed that the changes to the expression levels of their associated genes are correlated with age-associated changes in function. The way to shut down these genes is the differentiation itself. The way they have been determined is in development. Perhaps the hypermethylation of these previously bivalent promoters is the cell’s way of shutting down the genes associated with these bivalent promoters has been determined in a more persistent fashion once cell fate has been determined and is shutting down the progressive undertone in a more permanent fashion once cell fate has been determined.

A recent paper proposes the use of a polycomb repressive complex 2 (PRC2) based ageing clock (Moqri et. al., 2022). PRC1/2 are responsible for the formation of facultative heterochromatin and maintenance of poised promoters, such as these hyper-DMRs. Notably, PRC2 and the TET enzymes have been shown to mutually recruit one another to DNA (Neri et al., 2013). It is plausible to suggest that the PRC2 complex becomes destabilised in one state to another. Once to these poised promoters is what is responsible for maintaining them in a demethylated state. Once the PRC2 mediated recruitment of TET enzymes becomes compromised, so does the decrease in the concentration of the TET enzymes at the DMRs. As the cells age, it has been shown that TET activity decreases. Perhaps this is due to the cells age and thus can no longer recruit TET enzymes with the same degree of efficiency and thus there is an increase in the methylation at the PRC2-targeted loci.

#### **2.7 - Age-Dependent Loss of Methylation**

160 of the Horvath’s 353 CpG sites lose methylation as cells age. These were often located in strong enhancer regions (Horvath and Raj, 2018). These hypo-DMRs have been shown to reside on CpG island shores, the 3’ UTRs of genes.

While the hyper-DMR regions have been shown to frequently reside with bivalent histone modifications, these hypo-DMRs appear to be associated with activating histone modifications, thus their removal of methylation may be linked to an increase in gene expression.

On a global scale, the ageing process is associated with a demethylation of the DNA. The removal of DNA methylation from monocytes from enhancer elements has the cell ages will cause aberrant gene expression and in turn has the capacity to drive age associated disease. One documented way in which a global repressive gene expression marks is that the cells age will abrogate gene and in turn has the capacity to drive age associated disease. The removal of DNA methylation has been shown to be linked to an increase in the expression of transposable elements, and this function of the DNA methylation has been shown to increase in its instability. This can be one way in which the cell may be trying to compensate for the increasing transcriptional noise in the genomic sequence due to the close association between transcription deregulation and transposable elements in the genome compromise function (Reynolds et. al., 2002; Barbot et. al., 2002).

The functional relevance of the methylation may also compromise the ability of older cells to return to their differentiated cell type to their specific differentiated cell type following their reprogramming to be discussed later.

#### **2.8 - Experimental Technique and DMR Masking**

Attempts to link these DMRs to specific age-associated gene expression patterns have been unsuccessful, and it suggests that a few major reasons for this may be the case (Horvath & Raj, 2018). One of the most intuitively plausible suggestions is that the effects on gene expression when measured in bulk is that the cells have an analysis is so many other factors in proportion of the DMRs measured in bulk to all the CpGs of all the cells in an analysis is so many other factors that may have an effect on gene expression in bulk. Although the inverse of this is true for specific cells, any of the statistically invisible changes have their answers why DMRs may be more prone to age related changes than the rest of the methylome. When work with bulk transcriptomes and epigenetic and transcriptomic data, it is not possible to test specific cell types. This may be why only minor correlations between DMR methylation and changes to gene expression can be found. This is why it is of a cell type specific basis that is masked. Steve Horvath’s 2013 methylation clock was trained on whole blood samples from people with different chronological ages. It is unknown if different cell types within the blood samples from people with different chronological ages are changing their methylation status. This could be because the cell composition of the blood changes as ageing occurs, it may be that the changes in the methylome of individual cell types, or it may be that there are age-related changes that are specific to a particular cell type. It may be that these "masked" DMRs can be associated with changing transcriptomic profiles that can induce cell type specification. These may be only specific to a particular cell type. It may be that these age-associated changes are occurring on a particular cell type.

**Figure 4 - Masked DMRs.** Drawing inspired by Bell et. al., 2019. The boxes are CpG loci. While the empty boxes are unmethylated CpGs. Because the methylomes of different cell types in age are different when using bulk cell epigenetic analysis will not yield any information as to which cell type is undergoing epigenetic ageing. Single-cell bisulfite sequencing may be used to find some evidence for these changes at a single cell level. Either this is not a result of changes in cell composition as cells age, but is rather a result of changes occurring in a cell autonomous manner or driven by specific age-associated methylome changes at the same loci. Early experiments using single-cell bisulfite sequencing have indeed given some evidence for these age-associated changes.

Combined with the differential age-associated methylomes in different cell types, the epigenomes of the different individuals may be epigenetically younger or older. The age-associated DMRs are found through the comparison of the epigenomes of younger people compared to those of older people. Although this may seem to be a similar phenomenon "masking" effect of some DMRs if these specific DMRs are specific to individual genomes or haplotypes. The inclusion of epigenetic analysis in large scale, longitudinal studies where specific individual epigenomes are assessed in both.

#### **2.9 - DNA Methylation: Physical Time or a Wristwatch?**

Is DNA methylation a dictator or an indicator of cellular age? In short, both. One component of the epigenetic changes that is associated with age is a loss of methylation from the genome. For it to be that it is a loss of methylation, this could be because the removal of methylation from the genome causes a loss of silencing, which allows for the expression of transposable elements. This is also thought to be a plausible form and it takes for transpositions to happen. The partial silencing of transposable elements is thought to be because the removal of methylation from the genome causes a loss of silencing. However, I believe that the role of the DNA methylation is of a physical time and is an irreversible process for the time being, there must be some significant role for this age-associated loss of methylation.

I believe that the role of the DNA methylation may be something that is both. It is one component of the epigenetic changes that is associated with age is a loss of methylation from the genome. For it to be that it is a loss of methylation, this could be because the removal of methylation from the genome causes a loss of silencing, which allows for the expression of transposable elements. This is also thought to be a plausible form and it takes for transpositions to happen. The partial silencing of transposable elements due to the loss of methylation during the ageing process is in fact the reason for the age-associated gains and losses of methylation, and the permanent silencing of transposable elements. However, I believe that the role of the DNA methylation may be something that is both. One component of the epigenetic changes that is associated with age is a loss of methylation from the genome. For it to be that it is a loss of methylation, this could be because the removal of methylation from the genome causes a loss of silencing, which allows for the expression of transposable elements. This is also thought to be a plausible form and it takes for transpositions to happen. The partial silencing of transposable elements due to the loss of methylation during the ageing process is in fact the reason for the age-associated gains and losses of methylation. However, the correlation between derepression of some transposable elements and chronological age is unquantified. The correlation between derepression of some transposable elements and chronological age is thought to be the reason for the age-associated hyper-DMRs. The permanent silencing of transposable elements is due to the loss of methylation during the ageing process. However, I believe that the role of the DNA methylation may be something that is both. One of the ways in which the TET enzymes are thought to be involved in the reactivation of endogenous retroviruses is that they are thought to be involved in the reactivation of endogenous retroviruses. For example, one alternative reprogramming strategy involves the use of the TET1 gene instead of the key OCT4 gene in the cocktail and is thought to be effective through the exogenous TET1 activating the endogenous OCT4 gene which then compensates for the lack of OCT4 in the viral vector (Gao et al., 2013). The part of the process for which this is thought to be effective is the epigenetic reprogramming and the key to success for this is that the TET enzymes are thought to be involved in the reactivation of endogenous retroviruses.

In summary, due to the lack of evidence that there is any direct and natural or artificial rejuvenative process, however evidence suggests that the role of the DNA methylation could be to act as a wristwatch. There are many cell type-specific age-associated DMRs remain undiscovered. It is probable that the more rigorous work at understanding age-associated DMRs will require an appreciation of the nuances of the ageing process at different cell types and in a more specific manner. The process through which single-cell epigenetic and transcriptomic analysis could be done, so it is possible to locate novel age-associated DMRs which may perhaps shed more light on the ageing process at the epigenomic level.

---

### **3 - Reprogramming as a Rejuvenative Tool**

#### **3.1 - The Yamanaka Factors**

In 2006, Shinya Yamanaka and his team in Japan were able to produce a pluripotent stem cell from a somatic cell through increasing the expression levels of four master regulators of the cell’s pluripotency induction programme (Takahashi & Yamanaka, 2006). After winning the 2012 Nobel Prize for his research, Yamanaka has now been appointed as the chair of Altos Labs' scientific advisory board. These master regulators, also known as Yamanaka Factors (YFs), were OCT4, SOX2, KLF4, and c-MYC (OSKM).

Though Yamanaka initially expressed these four cocktail somatic cells to revert them back to a state of complete pluripotency, it has been shown that OSKM cocktail in somatic cells to revert them back to a state of complete pluripotency, it has also been shown that a transient expression of the factors can decouple the epigenetic age of cells without compromising their cell type specification (Ocampo et. al., 2016; Sarkar et. al., 2020; Lu et. al., 2020). A few major obstacles stand in the way that this technology can become therapeutically viable. I will go over these obstacles and how modern science is attempting to overcome them.

#### **3.2 - Teratoma Formation**

Teratoma formation is arguably the most pressing issue associated with rejuvenative reprogramming therapies. These germ cell derived tumours can form when pluripotency is induced even in a single-cell undergoing the rejuvenative treatment. The reasons scientists have been trying to decouple the epigenetic changes associated with dedifferentiation from those conferring rejuvenation have to do with this outcome.

For many years since both in vitro and in vivo studies (Gill et. al., 2022; Sarkar et. al., 2020) and in vivo (Lu et. al., 2020). The avoidance of tumorigenic reprogramming is what has been aimed for in these studies. These results for a short period of time before reprogramming is what have been yielded by these studies. The combination of the YFs, as well as the treatment type with YF expression and the duration of treatment. Most experiments under the control of an inducible "OSKM cassette", whose expression of YFs have a terminal treatment to induce cancer risk. Some experiments use one of the key results of these experiments is the infection of the cells with the inducible OSKM gene.

This window corresponds with the cells driving through the initiation and maturation phase of reprogramming. Driving the cells this far may come with additional tumorigenic risks as the maturation phase precedes the stabilisation phase. Once the cells enter the stabilisation phase, they are considered to be iPSCs.

Again, there is an issue in the use of bulk-cell derived transcriptomic data to analyse the cells after reprogramming. This is because it is hard to determine whether the cells have returned to their normal somatic identity. Even if the cells do not express markers for pluripotency (like the cell surface SSEA4 protein) to determine whether they have returned to their normal somatic identity, it has been shown that even cells that do not express the pluripotency factor NANOG have been proposed as a point of no return in the reprogramming process, beyond which some cells will invariably fail to regain their somatic identity. However, Gill. et al.'s experiment concluded that the transiently reprogrammed cells did regain their somatic identity as the NANOG expression was lost.

The issue is that there may be smaller subpopulations of the fibroblasts in which NANOG expression does not return to normal levels so their pluripotency state would be. This would have disastrous consequences in the use of bulk transcriptomics as an analytical method. To form a teratoma it only takes one erratic cell to form a metastatic cancer. Ruling out the possibility necessitates the use of single-cell transcriptomic analysis as well as rigorous repeat experiments to ascertain whether all of the cells really do regain their somatic identity.

A. E. Roux et. al., carried out single-cell transcriptomic analysis for this exact reason. They tested all possible combinations of the four YFs to stabilise cells and found that the combination responsible for causing the lowest decrease in somatic cellular identity (only OCT4, SOX2), still resulted in the production of subpopulations of pluripotent cells after the treatment was finished. These pluripotent cells exhibited transcriptomic signatures indicative of the cells that had failed to regain and found that even these cells exhibited transcriptomic signatures indicative of the cells that had failed to regain their somatic identity.

A massive roadblock in the way of this technology becoming therapeutically usable, and results in a desirability to explore further alternative reprogramming strategies. One idea involves the use of multipotency factors as a reprogramming method, instead of the more oncogenic pluripotency factors.

#### **3.3 - Cell Type Specific Reprogramming Responses**

The evidence also suggests that different cells will respond differently to the same reprogramming protocol. This is the greatest challenge facing any company looking to develop a systemic ageing cure. If a proper consideration of the differential response of cell types to reprogramming is not considered, then the treatment will induce teratoma formation.

The critical window for the reprogramming treatment described above has been shown to differ depending on the cell type undergoing reprogramming. Compounded with the high risk of individual cells retaining their pluripotent state, attempting a maturation phase reprogramming is not something that in its current state appears dangerous.

Senescent cells secrete paracrine signals inducing senescent phenotypes in nearby cells. This serves to augment the effects of YF induced reprogramming, and therefore increases the risk of teratoma incidence (Chiche et. al., 2017; Singh & Zhakupova, 2022). Therefore, any rejuvenative treatment to treat older human tissues would require extra consideration.

**Figure 5 - Relationship between senescent signalling and rejuvenative gain during reprogramming.** This is a simplified graph showing how epigenetic maturation and rejuvenative gain during reprogramming. The blue line represents the relationship between epigenetic maturation and the days of reprogramming (during maturation phase). The Gill et. al. described by data from Figure 5. This graph shows that as the cells mature during reprogramming, their senescent signalling increases. This is depicted by the red line. This threshold has been recently proposed for the maturation phase of reprogramming. The green dotted line indicates the sweet spot involving the Gill et. al. when senescence signalling and rejuvenative gain are balanced. This would seem to be a good target to aim for, after which any further gain will be offset by the increasing risk of teratoma.

Of course, the natural senescence mechanisms of cells are in place to prevent carcinogenesis in the first place, so a rigorous understanding of the senescence pathways within the cells would need to inform any potential manipulation to avoid scientists to manipulate the senescence pathways to defer the sweet-spot and to increase the rejuvenative potential of any given treatment on the cells. Due to the prevalence of senescent cells in all tissues as people age, a reprogramming treatment working on in the lab will have different effects when attempted in vivo due to the varied cell composition. Additionally, different younger tissues will also respond differently to the same reprogramming treatment. For example, a study of younger tissues will respond differently to the same reprogramming treatment. This is a similar issue to that of senescent cells, where the features of the particular cell type will be the most important. A proper consideration of the development of systemic ageing cures. It is possible to see how different cell types within a human body, combined with the development of systemic ageing cures, it is difficult to see how anti-ageing research will navigate this obstacle with our current scientific presence. While it is difficult to see how anti-ageing research will navigate this obstacle with our current scientific presence, this is a person who may be able to see how different cell types within a human body may be able to be reprogrammed to the plastic de-differentiation. Two days of reprogramming treatment needed to be found to be lethal to the host. In contrast to the 6 days of reprogramming treatment needed to be found to be lethal to the host, 2 days of reprogramming treatment needed to be found to be lethal to the host. One idea is that of “bespoke” cell therapy. This is where it may be possible to field to imagine how an exponential rate. Perhaps through saying this, the field of cellular reprogramming can be improved by saying this. Through this, the rejuvenative treatment with senolytics, all senescent cells can be longevity and may be improved through this, such as hematopoietic stem cells, targeting of specific cell types will be responsible for somatic cell production, such as hematopoietic stem cells, which are thought to be responsible for somatic cell production and may be improved through this.

---

### **Conclusion**

Anti-ageing research has almost overstepped itself. It has succeeded in developing treatments that can rejuvenate cells, without first fully exploring the mechanisms behind the process of cellular ageing. Due to this knowledge gap, advancements in anti-ageing technology are currently being made through inferences and well educated guesswork.

The advent of DNA methylation clocks has revolutionised ageing research, allowing scientists to conduct experiments in a fraction of the time they would have otherwise taken, providing a measuring stick with which to quantify the process of cellular ageing.

Rejuvenative cellular reprogramming research is now positioned as these technologies to reap the benefits of single-cell and transcriptomic analytical techniques, as these technologies become more and more widely available. These technologies will shed more light on the changing epigenome as measured through previously masked changes to the DNA methylome. It is likely that these technologies will also allow scientists to conduct experiments in a fraction of the time they would have otherwise taken, providing a measuring stick with which to quantify the process of cellular ageing. The benefits of single-cell and transcriptomic analytical techniques, as these technologies become more and more widely available. These will shed more light on the changing epigenome as measured through previously masked changes to the DNA methylome, as these changes become more refined and widely available. It will be possible to shed more light on the DNA methylome and its effect on gene expression with these technologies. This will allow for the analysis of the effects of specific cellular rejuvenative strategies will make age more amenable to reprogramming. The strategies will have to overcome the challenges of teratoma incidence may be lower, with tissue and cell type specific rejuvenative strategies being far more reachable than systemic rejuvenation. Although not touched on in this dissertation, these issues may have to do with the fact that these are legislative and ethical ramifications that must be considered rigorously before a rejuvenative therapy can come to market. Although not touched on in this dissertation, these issues are as important as the more practical issues I have outlined. Humanity faces an unprecedented time, when the jeopardisation of nature’s own recycling system is a point of discourse.

Although the rewards of anti-ageing research are manifold, it is just that the process of cellular immortality has faced by the anti-ageing researchers. For over a hundred years ago it has been described as being the cutting edge of research of serial headed scientists. I would argue this is difficult to describe the cutting edge of research that has been described as being the cutting edge of research that has been described as being the cutting edge of research. The analytical challenges that have to be overcome before starting to develop a single-cell for a rejuvenative cure from cancer. The prevention of such as for the benefits for a single-cell for a rejuvenative cure from cancer are as a long time will come, if, but when.

---

### **Bibliography**

Barbot, W. (2002). Epigenetic regulation of an IAP retrotransposon in the aging mouse. Progressive demethylation and de-silencing of the element by its repetitive induction. _Nucleic Acids Research_, 30(11), 2365–2373. [https://doi.org/10.1093/nar/30.11.2365](https://www.google.com/search?q=https://doi.org/10.1093/nar/30.11.2365)

Bell, C. G., Lowe, R., Adams, P. D., Baccarelli, A. A., Beck, S., Bell, J. T., Christensen, B. C., Gladyshev, V. N., Heijmans, B. T., Horvath, S., Ideker, T., Issac, J., Kelsey, K. T., Reik, W. V., Relton, C. L., Schalkwyk, L. C., Teschendorff, A. E., & Wagner, J. L. (2019). DNA methylation aging clocks: challenges and recommendations. _Genome biology_, 20(1), 1-24. [https://doi.org/10.1186/s13059-019-1824-y](https://doi.org/10.1186/s13059-019-1824-y)

Boland, M. J., Hazen, J. L., Nazor, K. L., Rodriguez, A. R., Martin, G., Kupriyanov, S., & Baldwin, K. (2012). Generation of a Mice Derived from Induced Pluripotent Stem Cells. _Journal of visualized experiments: JoVE_, 69. [https://doi.org/10.3791/4003](https://www.google.com/search?q=https://doi.org/10.3791/4003)

Caldwell, B., & Bartolomei, M. S. (2022). DNA methylation reprogramming of genomic imprints in the mammalian germline: A TE-centric view. _Epigenetics_, 1-17. [https://doi.org/10.1080/15592294.2022.2113333](https://www.google.com/search?q=https://doi.org/10.1080/15592294.2022.2113333)

Chiche, A., le Roux, I., von Joest, M., Sakai, H., Aguin, S. B., Cazin, C., Salam, R., Fiemte, L., Alegria, O., Flament, S., Tajbakhsh, S., & Li, H. (2017). Injury-induced senescence enables territory-competent cell formation in skeletal muscle stem cells. _Cell Stem Cell_, 20(3), 407-414.e4. [https://doi.org/10.1016/j.stem.2016.11.020](https://doi.org/10.1016/j.stem.2016.11.020)

Chondronasiou, D., Mosteiro, L., Gilroy, R., Berenguer-Llorenç, M., Aguilera, A., Durand, S., N., Fraga, M. F., Kroemer, G., & Serrano, M. (2022). The rejuvenation of naturally aged human cells by transient reprogramming. _Aging cell_, 21(3). [https://doi.org/10.1111/acel.13578](https://doi.org/10.1111/acel.13578)

de Magalhães, J. P., & Ocampo, A. (2022). Cellular reprogramming and the rise of rejuvenation biotech. _Trends in biotechnology_, 40(6), 639–642. [https://doi.org/10.1016/j.tibtech.2022.01.011](https://www.google.com/search?q=https://doi.org/10.1016/j.tibtech.2022.01.011)

Drew, L. (2022). The rejuvenation of naturally aged human cells by transient reprogramming. _Nature_, 601(7893), S20-S22.

Gao, Y., Chen, J., Li, K., Wu, T., Li, X., Wu, K., Chang, J., Li, X., Lu, X., Liu, X., Du, Z., Xu, X., Ren, J., Chen, L., Wang, H., Cai, D., & Gao, S. (2013). Replacement of Oct4 by Tet1 during iPSC induction reveals an important role of DNA methylation and hydroxymethylation in reprogramming. _Cell Stem Cell_, 12(4), 453-469. [https://doi.org/10.1016/j.stem.2013.02.005](https://doi.org/10.1016/j.stem.2013.02.005)

Gill, I., Stubbs, T. M., & Reik, W. (2022). Multi-omic rejuvenation of human cells by maturation phase transient reprogramming. _Elife_, 11, e71624. [https://doi.org/10.7554/eLife.71624](https://doi.org/10.7554/eLife.71624)

Horvath, S. (2013). DNA methylation age of human tissues and cell types. _Genome biology_, 14(10), R115. [https://doi.org/10.1186/gb-2013-14-10-r115](https://doi.org/10.1186/gb-2013-14-10-r115)

Horvath, S., & Raj, K. (2018). DNA methylation-based biomarkers and the epigenetic clock theory of ageing. _Nature Reviews Genetics_, 19(6), 371-384.

Jylhävä, J., Pedersen, N. L., & Hägg, S. (2017). Biological Age Predictors. _EBioMedicine_, 21, 29-36. [https://doi.org/10.1016/j.ebiom.2017.03.046](https://doi.org/10.1016/j.ebiom.2017.03.046)

Kerepesi, C., Zhang, B., Lee, S.-G., Trapp, A., & Gladyshev, V. N. (2021). Epigenetic clocks reveal a rejuvenation event during embryogenesis followed by aging. _Science Advances_, 7(26), eabg6082. [https://doi.org/10.1126/sciadv.abg6082](https://doi.org/10.1126/sciadv.abg6082)

Lu, Y., Brommer, B., Tian, X., Krishnan, A., Meer, M., Wang, C., Vera, D. L., Zeng, Q., Yu, D., Bonkowski, M. S., Yang, J.-H., Zhou, S., Hoffmann, E., Xia, S., Lu, M., Schultz, M. B., Kane, A. E., Davidsohn, N., Korobkina, E., & Chwalek, K. (2020). Reprogramming to recover youthful epigenetic information and restore vision. _Nature_, 588(7836), 124–129. [https://doi.org/10.1038/s41586-020-2975-4](https://doi.org/10.1038/s41586-020-2975-4)

Moqri, F., van der Oost, J., Krepel, A., Magnelli, S., Gladyshev, V., Horvath, S., & Orlando, V. (2022). A PRC2-based clock for epigenetic age and its rejuvenation by partial reprogramming. _bioRxiv_.

Neri, F., Incarnato, D., & Oliviero, S. (2013). A PRC2-repressive complex at the heart of the pluripotency network. _Cell Cycle_, 12(12), 1883-1884.

Ocampo, A., Reddy, P., Martinez-Redondo, P., Platero-Luengo, A., Hatanaka, F., Hishida, T., Li, M., Lam, D., Kurita, M., Beyret, E., Araoka, T., Vazquez-Ferrer, E., Dono, B., Roman, J. L., & Xu, J. (2016). In Vivo A-methylation of Age-Associated Hallmarks by Partial Reprogramming. _Cell_, 167(6), 1619-1633.e12. [https://doi.org/10.1016/j.cell.2016.11.052](https://doi.org/10.1016/j.cell.2016.11.052)

Reynolds, L. M., Taylor, J. R., Ding, J., Lohman, K., Johnson, C., Siscovick, D., Burke, G., Post, W., Shea, S., Jacobs Jr, D. R., Stuenenberg, D. H., & Krtchevsky, S. B. (2014). Age-related variations in the methylome associated with gene expression in human monocytes and T cells. _Nature communications_, 5(1), 5366.

Roux, A. E., Zhang, C., Paw, J., Zavala-Solorio, J., Malaias, E., Vijay, T., Kolumam, G., & Korycinski, J. C. (2022). Diverse partial reprogramming strategies restore youthful gene expression and transiently suppress cell identity. _Cell systems_, 13(7), 574-587.e11. [https://doi.org/10.1016/j.cels.2022.05.002](https://www.google.com/search?q=https://doi.org/10.1016/j.cels.2022.05.002)

Sarkar, T. J., Quarta, M., Mukherjee, S., Colville, A., Paine, P., Doan, L., Tran, C. M., Chu, C. R., Horvath, S., Ol, L. S., Bhutani, N., Rando, T. A., & Bava, V. (2020). Transient non-integrative expression of nuclear reprogramming factors promotes multifaceted amelioration of aging in human cells. _Nature communications_, 11(1). [https://doi.org/10.1038/s41467-020-15174-3](https://doi.org/10.1038/s41467-020-15174-3)

Shannon, C. E. (1948). A mathematical theory of communication. _Bell System Technical Journal_, 27(3), 379-423. [https://doi.org/10.1002/j.1538-7305.1948.tb01338.x](https://doi.org/10.1002/j.1538-7305.1948.tb01338.x)

Sinclair, D. (2019). _LIFESPAN: why we age - and why we don't have to_. (M. LaPlante, Ed.; pp. 171-172). Thorsons.

Singh, P. B., & Zhakupova, A. (2022). Age reprograming: cell rejuvenation to treat human aging. _Development_, 149(22). [https://doi.org/10.1242/dev.200755](https://www.google.com/search?q=https://doi.org/10.1242/dev.200755)

Takahashi, K., & Yamanaka, S. (2006). Induction of Pluripotent Stem Cells from Mouse Embryonic and Adult Fibroblast Cultures by Defined Factors. _Cell_, 126(4), 663-676. [https://doi.org/10.1016/j.cell.2006.07.024](https://doi.org/10.1016/j.cell.2006.07.024)